ros_ign_bridge/test/serverclient/ros_client.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <thread>
+
+#include "rclcpp/rclcpp.hpp"
+
+#include "ros_ign_interfaces/srv/control_world.hpp"
+
+using namespace std::chrono_literals;
+
+/////////////////////////////////////////////////
+TEST(ROSClientTest, WorldControl)
+{
+  rclcpp::init(0, NULL);
+  auto node = std::make_shared<rclcpp::Node>("test_ros_client_to_ign_service");
+  auto client = node->create_client<ros_ign_interfaces::srv::ControlWorld>(
+    "/ign_ros/test/serviceclient/world_control");
+  std::this_thread::sleep_for(1s);
+  client->wait_for_service();
```

Comment:
We should probably have a timeout here to avoid blocking forever in case there's an problem.

Reply:
Done in https://github.com/ignitionrobotics/ros_ign/pull/211/commits/b2a37daabf00db7c9bfed437d7d096dc535f597a